app/database/models/tasks_list.py

```diff
@@ -5,6 +5,14 @@
 
 
 class TasksListModel(db.Model):
+    """Model representation of a list of tasks.
+    
+    Attributes:
+        id: Id of the list of tasks.
+        tasks: A list of tasks.
```

Comment:
```suggestion
        tasks: A list of tasks, using JSON format.
```

Reply:
I just wanted to make this clear, because of it is an important detail of tasks implementation
cc @ramitsawhney27